doc/boot_storage_needed_info.md

```diff
@@ -0,0 +1,38 @@
+List of Information Needed About Storage Configuration
+======================================================
+
+Mount Points
+---------------
+
+- what partition is mounted to "/" ( so it can be queried )
```

Comment:
ok, I will replace it with generic device as I know that it can be whole disk. On other hand bootloader is not much interested in filesystem, but more what kind of device it is.